build.gradle

```diff
@@ -56,14 +56,14 @@ dependencies {
     implementation("com.google.code.gson:gson:2.8.6")
     implementation("com.google.guava:guava:30.1.1-jre")
     implementation(objectify)
-    implementation("com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20200615.1")
+    implementation("com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20211018.+")
     implementation("com.mailjet:mailjet-client:4.5.0")
     implementation("com.sendgrid:sendgrid-java:4.6.0")
     implementation("com.sun.jersey:jersey-client:1.19.4")
     implementation("com.sun.jersey:jersey-core:1.19.4")
     implementation("com.sun.jersey.contribs:jersey-multipart:1.19.4")
     implementation("javax.servlet:javax.servlet-api:3.1.0")
-    implementation("org.apache.solr:solr-solrj:8.8.1")
+    implementation("org.apache.solr:solr-solrj:8.+")
```

Comment:
Use exact versions instead of wildcards. Right now this probably resolves to `8.11.0`, but we want to stay as close as possible (i.e. `8.8.2`) just in case there are regressions across minor versions.

Reply:
done